README.md

```diff
@@ -83,6 +90,7 @@ The following options are configurable:
     page scrape request, this is the amount of time (in milliseconds) that the
     code will sleep prior to re-issuing the request. Defaults to
     <code>1000</code> (ms).</li>
+    <li><code>keys</code> : An object of API keys required by services dependent upon keys. <strong>Current services requiring an API key to operate: Best Buy.</strong> Keys can also be provided as environment variables, in the form of <code><em>SERVICE</em>_KEY</code>, such as <code>BESTBUY_KEY</code>.</li>
```

Comment:
This is very minor, but do you think it would be worthwhile to put the key name in this description for Best Buy (`bestbuy` in the example above)?  For now it's just the one service but I'm wondering if we can use this as an opportunity to list the acceptable key names here.